### PR TITLE
Use dependencies.io Lerna support & disable issue creation

### PR DESCRIPTION
**Pull request by Hypnosphi**
See https://github.com/Hypnosphi/storybook/pull/85

### CODE REVIEW

**Comment by ndelangen** [comment]
why based on 3.3 ?

**Comment by Hypnosphi** [comment]
master doesn't have `yarn.lock`

**Comment by Hypnosphi** [comment]
After a discussion in slack, I disabled issue creators:

```
hypnos [2:52 AM] 
Do we want issues for major updates in deps? https://github.com/Hypnosphi/storybook/issues
 

norbert 
[2:56 AM] 
what do you mean?


hypnos [3:06 AM] 
The recommended setup for `dependencies.io` is PRs for minors and patches, issues for majors (edited)


shilman 
[3:28 AM] 
sounds a little heavy-handed to me, but that’s just personal preference


[3:28] 
i wouldn’t do issues if it was up to me, but i am also open to being pursuaded (edited)


dduan
[5:47 PM] 
I think it just creates unnecessary spam and nothing actionable. Perhaps a few people want to own upgrading dependencies and be assigned those issues? We have enough open tickets already.
```

File: dependencies.yml [hunk 1]
```diff
@@ -89,195 +30,29 @@ collectors:
...
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/links
+- type: js-lerna
+  path: /
+  settings:
+    bootstrap_command: yarn && yarn bootstrap -- --core
```
Comment (ndelangen) [inline dependencies.yml]: I think we want to hold this until #1810

Comment (Hypnosphi) [reply dependencies.yml]: Ok, I'll add both to 3.3 milestone (see https://github.com/storybooks/storybook/pull/1803#issuecomment-327573050)

Comment (Hypnosphi) [reply dependencies.yml]: In fact, the command has to stay the same: the bootstrap script won't work without root depandencies

**Comment by ndelangen** [comment]
Ok, just to get everything clear:

- Merging this would initially do nothing, since the tool would only trigger on `master` right?
- This would replace greenkeeper completely, after this lands on master, we should disable greenkeeper.

**Comment by Hypnosphi** [comment]
Both true

**Comment by Hypnosphi** [comment]
I don't think I understand what's going on here: https://circleci.com/gh/storybooks/storybook/12142?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by ndelangen** [comment]
I think the best way to proceed with this, is to get yarn-workspaces and yarn lock files on master and get this merged to master.

What do you think @Hypnosphi @shilman @igor-dv @storybooks/team ?

**Comment by Hypnosphi** [comment]
I will do that as long as everyone is OK with it

**Comment by danielduan** [comment]
seems reasonable to me. This is a developer oriented change so merging to master wouldn't really affect the product itself.

**Comment by ndelangen** [comment]
@Hypnosphi Can you make the PR? I think it's a go.

**Comment by Hypnosphi** [comment]
Should it be one PR for `yarn` and then another one for `dependencies.io`?

**Pull request closed without merging**